pkg/server/config.go

```diff
@@ -368,6 +368,11 @@ func NewConfig(opts *kcpserveroptions.CompletedOptions) (*Config, error) {
 
 		if kcpfeatures.DefaultFeatureGate.Enabled(kcpfeatures.SyncerTunnel) {
 			apiHandler = tunneler.WithSyncerTunnel(apiHandler)
+			apiHandler = WithPodSubresourceProxying(
```

Comment:
@ncdc @sttts @davidfestal when TMC is "broken out" of the core API server, how does this work?

Reply:
Would we want to include this proxying in kcp core anyways, because it's "easier"?

Could/would we want to move this to the front proxy?

Reply:
yes, front-proxy most likely